test/remote-connector.test.js

```diff
@@ -87,3 +91,55 @@ describe('RemoteConnector', function() {
     });
   });
 });
+
+describe('Custom Path', function() {
+
+  var test = this;
+
+  before(function(done) {
+    
+    var db_config = {
+      connector: loopback.Memory,
+      name: 'db'
+    };
+    var db = loopback.createDataSource('db', db_config);
+
+    var ServerModel_config = {
+      dataSource: 'db',
+      http: {
+        path: '/custom'
+      }
+    };
+
+    var ServerModel = loopback.PersistedModel.extend('TestModel', {foo: 'string'}, ServerModel_config);
+    ServerModel.attachTo(db);
+
+    server = test.server = createAppWithRest();
+    server.model(ServerModel);
+    server.dataSource(db, db_config);
+
+    done();
+
+  });
+
+  it('should support http.path configuration', function(done) {
+    var server = test.server;
+
+    listenAndSetupRemoteDS(test, server, 'remoteDS', function() {
```

Comment:
I am proposing to call `listenAndSetupRemoteDS` from the `before` hook above. Unless there is a good reason why it's a bad idea?